src/components/CardView.js

```diff
@@ -8,17 +8,18 @@ const zoomLevel = 13;
 
 class Card extends Component {
     render() {
-        const center = [51.505, -0.09]
+        const defaultLatitude = 47.71780751;
+        const defaultLongitude = 8.666430535;
         const {gpsData} = this.props;
-        const start = gpsData[0];
-        const end = gpsData[gpsData.length - 1];
+        let start = gpsData[0];
+        let end = gpsData[gpsData.length - 1];
         const array = [];
         gpsData.map(element => {
             array.push([element.latitude, element.longitude]);
         });
-        console.log(array);
-        console.log(start);
         const data = [[51.505, -0.09], [51.51, -0.1], [51.51, -0.12]];
+        start === undefined ? (start = [], start.longitude=defaultLongitude, start.latitude=defaultLatitude) : (start = gpsData[0]);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove use of all comma operators in this expression. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS878)